FogCreek/Glitch-Community
# Husky calf: componentize header

## Links
* Remix link: https://husky-calf.glitch.me/
* Manuscript link: https://glitch.manuscript.com/f/cases/3328391/

## GIF/Screenshots:
![Screen Shot 2019-04-25 at 1 11 36 PM](https://user-images.githubusercontent.com/938722/56754712-b2258900-675b-11e9-86b0-8a14849469a4.png)
![Screen Shot 2019-04-25 at 1 08 04 PM](https://user-images.githubusercontent.com/938722/56754827-f6188e00-675b-11e9-943f-6f2c9985c9e4.png)

## Changes:
* rewrite Header & Logo as components
* add dayJS config & user prefs/dev toggles contexts to stories
* move header margins, NewStuffContainer into layout

## How To Test:
* visit  https://husky-calf.glitch.me/
* the header should render correctly, with "New Project" and "Sign in" menu buttons
* log in by setting `localStorage.cachedUser`
* the header should render with "New Project", "Resume Coding" and user menu buttons
